### PR TITLE
DPL Analysis: cleanups in TreeToTable code

### PR DESCRIPTION
**Pull request by aalkin**

### CODE REVIEW

**Comment by ktf** [comment]
Please fix the conflict.

File: Framework/Core/src/TableTreeHelpers.cxx [hunk 1]
```diff
@@ -228,13 +228,11 @@ std::pair<std::shared_ptr<arrow::ChunkedArray>, std::shared_ptr<arrow::Field>> B
     std::unique_ptr<TBufferFile> offsetBuffer;
 
     uint32_t offset = 0;
-    uint32_t lastOffset = offset;
+    uint32_t lastOffset;
```
Comment (ktf) [inline Framework/Core/src/TableTreeHelpers.cxx:231]: Why not initialising?

**Comment by ktf** [comment]
Tested on hyperloop. I am not sure the idea of leaving some variable uninitialised is actually a good idea, but we can fix it separately.

File: Framework/Core/src/TableTreeHelpers.cxx (hunk 1 above)
Comment (aalkin) [reply Framework/Core/src/TableTreeHelpers.cxx:231]: It is redundant, as it is always reset in a loop, before it is used, and if loop is not executed it is not used.

Comment (ktf) [reply Framework/Core/src/TableTreeHelpers.cxx:231]: It's  redundant until we change the code and then we an hard to spot random memory access. I would always initialise  things.

**Pull request merged**